src/types/Status.ts

```diff
@@ -0,0 +1,5 @@
+export enum Status {
+  all = 'All',
+  active = 'Active',
+  completed = 'Completed',
+}
```

Comment:
please write with first upperCase letter
`active` to `Active`

![image](https://github.com/mate-academy/react_todo-app-with-api/assets/91946036/2fac4390-52cf-44bc-8ddc-c5f473ba0ed7)